src/core/services/dom.d.ts

```diff
@@ -0,0 +1,5 @@
+export declare function css(element: object, property: string, value: string): any;
```

Comment:
element maybe should be DomElement or HtmlElement do we have that is standart definitions?